roles/matrix-dynamic-dns/defaults/main.yml

```diff
@@ -0,0 +1,40 @@
+# Whether dynamic dns is enabled
+matrix_dynamic_dns_enabled: false
```

Comment:
We usually set `_enabled` to `true` in each role, so that people using our roles outside of the playbook have them enabled without doing anything.

For the playbook itself, we enable/disable roles by overriding such `_enabled` variables in `group_vars/matrix_servers`.

Reply:
I've updated this line. I think the life we want is to only enable this when there is work to be done. i.e. that there exists nonzero domains to set dynamically. Check out this update!